style.css

```diff
@@ -183,6 +183,8 @@ h2 {
   box-sizing: border-box;
   grid-column: 1/-1;
   grid-row: 2/3;
+  background: #ebf0ee;
+  min-height: 220px;
 }
 
 .work-thumbnail img {
```

Comment:
Good job fixing the contract ratio 👍
- [ ] Wonderful job adding color to each element in your portfolio 🥇. Let's now fix a few elements having less color contrast. This is because there are some people who cannot read the text if there is not sufficient contrast between the text and the background, as an accessibility requirement, please make sure that your page has the correct contrast ratio 👍. You can use [this aditus tool](https://www.aditus.io/button-contrast-checker/) to help you to check the color contrast 💪
 - [ ] Alternatively you can also use **_WCAG Contrast checker_** or **_WebDev Toolbar_** (for windows)

### **_From WebDev Toolbar 👇_**
![image](https://user-images.githubusercontent.com/106119105/214452178-aa96402a-a480-4792-b38b-a72a3995c4f5.png)


### **_From WCAG Contrast checker 👇_**
![image](https://user-images.githubusercontent.com/106119105/214451938-a60f91fe-69de-4e2d-9af4-e6a5aa083845.png)


**_NOTE: Please note that this applies to you entire `style.css` file. I'm adding it here because I'm unable to select the content above these lines. I know you can fix this. Right ? 🎯_**